recipe/meta.yaml

```diff
@@ -53,7 +53,6 @@ about:
   license_file:
     - LICENSE
     - notebook/labextension/static/third-party-licenses.json
-    - notebook/static/third-party-licenses.json
```

Comment:
Normally this file should still exist in the upstream tarball. But no strong opinion on whether we should keep it or not in the recipe.

Reply:
ah, let's try

Reply:
[24e5ba6](https://github.com/conda-forge/notebook-feedstock/pull/155/commits/24e5ba669cd979ebe7fcd083d10e3635978d0010)